code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -4391,6 +4391,14 @@
 	density = 2.09
 	specheatcap = 1.65
 
+/datum/reagent/sodiumchloride/reaction_turf(var/turf/simulated/T, var/volume)
+	if(..())
+		return 1
+
+	if(volume >= 50 && !(locate(/obj/effect/decal/cleanable/salt) in T))
+		new /obj/effect/decal/cleanable/salt(T)
```

Comment:
no density checks of any kind, appears to let you create piles of salt on walls and under machines